python/spacewalk/satellite_tools/reposync.py

```diff
@@ -38,48 +40,58 @@
 from rhn.stringutils import ustr
 
 from uyuni.common.usix import raise_with_tb
-
-from spacewalk.server import rhnPackage, rhnSQL, rhnChannel, suseEula
 from uyuni.common import fileutils
-from spacewalk.common import rhnLog, rhnMail, suseLib
-from spacewalk.common.rhnTB import fetchTraceback
-from spacewalk.common import repo
 from uyuni.common.rhnLib import isSUSE, utc
 from uyuni.common.checksum import getFileChecksum
 from uyuni.common.context_managers import cfg_component
+
+from spacewalk.server import rhnPackage, rhnSQL, rhnChannel, suseEula
+from spacewalk.common import rhnLog, rhnMail, suseLib
+from spacewalk.common.rhnTB import fetchTraceback
+from spacewalk.common.repo import GeneralRepoException
```

Comment:
Notable but easy to miss change; we were importing the full `repo` module but it seems that we were using only the exception from it.

I made this more specific because some methods use the `repo` name and shadow this import.